docs/deploying_apps/deploying_django.md

```diff
@@ -1,132 +1,109 @@
-### Ignoring files
+This section explains how to deploy an app using the Django framework. You may also need to refer to the [Cloud Foundry documentation about the Python buildpack](https://docs.cloudfoundry.org/buildpacks/python/index.html).
 
-1. Add `*.pyc` and `local_settings.py` to your `.gitignore`.
-1. [Exclude your `.gitignore`d files from Cloud Foundry.](/deploying_apps/excluding_files/)
+Note that the only database service currently supported by PaaS is PostgreSQL. If your Django app requires a database, it must be able to work with PostgreSQL.
 
-### The Python runtime
+These steps assume you have already carried out the setup process explained in the [Quick Setup Guide](/getting_started/quick_setup_guide) section.
 
-Next, you want to tell Cloud Foundry which Python runtime to use. To do this, create a `runtime.txt` file, and put the full version of Python you want to deploy with in it. For instance,
+If you are just getting started learning CloudFoundry, you can use the sandbox space by running: ``cf target -o sandbox``
 
-```
-python-3.4.1
-```
+1. Check out your Django app to a local folder.
 
-### `requirements.txt`
+2. Add `*.pyc` and `local_settings.py` to your `.gitignore`, file, then 
+   [exclude files ignored by Git](/deploying_apps/excluding_files/) so Cloud Foundry will ignore them too.
 
-There are a couple of packages used for the deploy that will be helpful. You'll want to add them to the `requirements.txt` for your project.
+3. Tell Cloud Foundry which Python runtime to use by creating a `runtime.txt`   file in the root of the local folder. The contents of the file should  
+   be:
 
-Below are the lines I added to my `requirements.txt` file. Later versions will probably work but haven't been tested.
+        python-3.4.1  
 
-```
-whitenoise==1.0.6  #manages static assets
-dj-database-url==0.3.0 #grabs environment variables and dumps them into a Django settings file
-waitress==0.8.9 #a pure python WSGI server that is a replacement for gunicorn
-```
+    replacing "3.4.1" with the version of Python you want to use.
```

Comment:
...and which is supported by the python buildpack. Which you can find out by looking at buildpack version in `cf buildpacks` (e.g. python_buildpack-cached-v1.5.5.zip) and then checking release notes for that version: https://github.com/cloudfoundry/python-buildpack/releases